src/ansys/aedt/core/workflows/hfss/choke_designer.py

```diff
@@ -0,0 +1,505 @@
+
+                if category in selected_options:
+                    selected_options[category].set(selected_option)
+
+    def update_entries():
+        for category, options in config_dict.items():
+            for field, value in options.items():
+                entry_widget = entries_dict.get((category, field))
+
+                if entry_widget:
+                    entry_widget.delete(0, tkinter.END)
+                    entry_widget.insert(0, str(value))
+
+    button_frame = ttk.Frame(master, style="PyAEDT.TFrame", relief=tkinter.SUNKEN, borderwidth=2)
+    button_frame.pack(fill=tkinter.X, pady=0)
+
+    save_button = ttk.Button(
+        button_frame, text="Save Configuration", command=save_configuration, style="PyAEDT.TButton"
+    )
+    load_button = ttk.Button(
+        button_frame, text="Load Configuration", command=load_configuration, style="PyAEDT.TButton"
+    )
+    change_theme_button = ttk.Button(button_frame, text="\u263D", command=toggle_theme, style="PyAEDT.TButton")
+    save_button.pack(side=tkinter.LEFT, padx=5)
+    load_button.pack(side=tkinter.LEFT, padx=5)
+    change_theme_button.pack(side=tkinter.RIGHT, padx=5, pady=40)
+
+    def callback():
+        master.flag = True
+        if validate_configuration(config_dict):
+            master.destroy()
+
+    export_hfss = ttk.Button(button_frame, text="Export to HFSS", command=callback, style="PyAEDT.TButton")
+    export_hfss.pack(side=tkinter.LEFT, padx=5)
+
+    tkinter.mainloop()
+
+    choke_config = {}
+    if master.flag:
+        choke_config = {"choke_config": config_dict}
+    return choke_config
+
+
+def main(extension_args):
+    choke_config = extension_args["choke_config"]
+
+    app = ansys.aedt.core.Desktop(
+        new_desktop=False,
+        version=version,
+        port=port,
+        aedt_process_id=aedt_process_id,
+        student_version=is_student,
+    )
+
+    active_project = app.active_project()
+
+    hfss = None
+    if not active_project:
+        hfss = Hfss()
+        hfss.save_project()
+        active_project = app.active_project()
+
+    active_design = app.active_design()
+
+    project_name = active_project.GetName()
+
+    design_name = None
+    if active_design:
+        design_name = active_design.GetName()
+
+    if not hfss:  # pragma: no cover
+        hfss = Hfss(project_name, design_name)
+
+    hfss.solution_type = "Terminal"
+
+    # Create temporary directory for JSON file
+    temp_dir = Path(tempfile.mkdtemp())
+    json_path = temp_dir / "choke_params.json"
+
+    write_configuration_file(choke_config, str(json_path))
+
+    # Verify parameters
+    dictionary_values = hfss.modeler.check_choke_values(str(json_path), create_another_file=False)
+
+    # Create choke geometry
+    list_object = hfss.modeler.create_choke(str(json_path))
+
+    # Get core and winding objects
+    _ = list_object[1]
+    first_winding_list = list_object[2]
```

Comment:
```suggestion
    # Get winding objects
    first_winding_list = list_object[2]
```
What is the point of accessing list_object[1] here ?

Reply:
Not needed indeed